Add expanded menu

orkosinha opened the pull request:
Added an expandedMenu field to the CampusEateryType.

orkosinha committed:
Add expandedItems parsing

orkosinha committed:
Fix Hound issues

ilikesymmetry reviewed (changes requested):
all in all, nice addition.  just a few small things to change 👍

orkosinha committed:
Make suggested changes

- Added comments
- Fixed ordering of imports
- Rename options to choices
- New constant for DiningStack

ilikesymmetry reviewed (commented):
lgtm :)

young-k reviewed (approved):
#shipit, let's add a separate PRs to use lambdas because I think it's more concise and clean to read @orkosinha
